lib/kennel/models/record.rb

```diff
@@ -77,7 +77,7 @@ def ignore_default(expected, actual, defaults)
         end
       end
 
-      attr_reader :project, :unfiltered_validation_errors, :filtered_validation_errors, :as_json
+      attr_reader :project, :unfiltered_validation_errors, :as_json
```

Comment:
If the Record doesn't really know about filtering, then why is this called "unfiltered" validation errors?

Reply:
renamed to validation_errors